resources/views/processes/screens/edit.blade.php

```diff
@@ -58,6 +79,7 @@
     <script>
         new Vue({
             el: '#editGroup',
+            mixins: addons,
```

Comment:
Is this right? Mixins should be an array of JS objects; I don't see where `addons` is defined.

Reply:
yes, it's arming with a controller and passed that way.